### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by ibiimemon** [emanuelecaurio/react-flip-card#4]
Error in function ReactFlipCard in ./node_modules/reactjs-flip-card/dist/ReactFlipCard.js:34
Error in gatsby js

  32 |             setIsFlipped(flipByProp);
  33 |     }, [flipByProp]);
> 34 |     return (React.createElement("div", { style: containerStyle, className: `${cssMod.container} ${containerCss} `, onMouseEnter: (e) => {
     |                                                                               ^
  35 |             if (flipTrigger === FlipTrigger.onHover) {
  36 |                 setIsFlipped(!isFlipped);
  37 |             }

**Comment by marksteven** [comment]
Yes I get this too and also Gatsby(React)
Hope this helps.

One unhandled runtime error found in your files. See the list below to fix it:


Error in function ReactFlipCard in ./node_modules/reactjs-flip-card/dist/ReactFlipCard.js:34
Cannot read properties of undefined (reading 'container')

./node_modules/reactjs-flip-card/dist/ReactFlipCard.js:34
Open in Editor
  32 |             setIsFlipped(flipByProp);
  33 |     }, [flipByProp]);
> 34 |     return (React.createElement("div", { style: containerStyle, className: `${cssMod.container} ${containerCss} `, onMouseEnter: (e) => {
     |                                                                              ^
  35 |             if (flipTrigger === FlipTrigger.onHover) {
  36 |                 setIsFlipped(!isFlipped);
  37 |             }

**Comment by emanuelecaurio** [comment]
Yeah, it seems there are difference between CRA and Gatsby about importing css modules.
@ibiimemon @marksteven I suggest to edit your `gatsby-config.js` as suggested here:
https://www.reddit.com/r/gatsbyjs/comments/lzln0w/comment/gq3l7eu/?utm_source=share&utm_medium=web2x&context=3

Let me know if it works. 
If yes, I'll update the doc

**Comment by emanuelecaurio** [comment]
@ibiimemon   @marksteven I just removed modules.css for Gatsby and CRA compatibility in the last version 2.0.3. 
Let me know if there are any issues
